src/build_tools/hooks_hydra.py

```diff
@@ -72,6 +72,20 @@
 SUFFIX_MODULES_SYMLINK = 'all'
 
 
+def get_group(name, version):
+    "get the user group for licensed software"
+    group = None
+    if name in SOFTWARE_GROUPS:
+        if isinstance(SOFTWARE_GROUPS[name], str):
+            group = SOFTWARE_GROUPS[name]
+        else:
+            for versionsuffix, grp in SOFTWARE_GROUPS[name].items():
+                if version.startswith(versionsuffix):
+                    group = grp
+                    break
```

Comment:
```suggestion
        group = SOFTWARE_GROUPS[name]
        if not isinstance(group, str):
            version_major = version.split('.')[0]
            try:
                group = group[version_major]
            except KeyError:
                raise EasyBuildError("No group defined for this version of licensed sofwtare")
```

Reply:
`version.split('.')[0]` is not the same as `version.startswith(versionsuffix)`, and both have issues:
- not all versions are formatted as `major.minor`
- `startswith` cannot distinguish between e.g. versions `6` and `61`

so i've changed it to use regexes, which provide the most flexibility and accuracy.
https://github.com/vub-hpc/build_tools/pull/43/commits/7cc44ecfe36a49a0edea4cb1a75e4359ba05cbd1